pkg/hintrunner/zero/zerohint_memcpy_test.go

```diff
@@ -0,0 +1,51 @@
+package zero
+
+import (
+	"testing"
+
+	"github.com/NethermindEth/cairo-vm-go/pkg/hintrunner/hinter"
+	"github.com/NethermindEth/cairo-vm-go/pkg/utils"
+)
+
+func TestMemcPyHint(t *testing.T) {
+	runHinterTests(t, map[string][]hintTestCase{
+		"MemcPyContinueCopying": {
+			{
+				operanders: []*hintOperander{
+					{Name: "continue_copying", Kind: uninitialized},
+				},
+				ctxInit: func(ctx *hinter.HintRunnerContext) {
+					ctx.ScopeManager.EnterScope(map[string]any{})
+					err := ctx.ScopeManager.AssignVariables(map[string]any{
+						"n": uint64(1),
```

Comment:
Can we add one more test with `"n": uint64(0)`

Reply:
sure!

Reply:
I found that the uint64 type only supports this range of numbers: 
```go
// uint64 is the set of all unsigned 64-bit integers.
// Range: 0 through 18446744073709551615.
type uint64 uint64

```

can this be another reason to use felt instead of uint ?

Reply:
@TAdev0 @har777 Look this [comment](https://github.com/NethermindEth/cairo-vm-go/pull/359/#discussion_r1588858507) from @cicr99

Reply:
taking that into consideration adding a test where `"n": uint64(0)` is not going to retrieve the desired value in this moment.